tests/acceptance/features/webUIFilesSearch/search.feature

```diff
@@ -59,38 +59,6 @@ Feature: Search
     #And file "lorem-big.txt" with path "/strängé नेपाली folder" should be listed in the search results in the other folders section on the webUI
     #But file "lorem.txt" with path "/simple-folder" should not be listed in the search results in the other folders section on the webUI
```

Comment:
Most of the other scenarios are also commented. Maybe they can be removed too? Do they even work as expected? We also have some search-related tests for e2e as well

Reply:
Those commented steps are not implemented and we are adding related tests in e2e so these steps can be removed.